content.md

```diff
@@ -185,7 +185,7 @@ Tap on form field and type the desired value using the pop-up keyboard.
 
 ### 5.2 Auto LookUp
 
-Start filling in the first 3 letters of the parameter(eg. Member, Garden ect.) you want to add to the form field. Select the relevant item from the list displayed below the text field. If this parameter(eg. Member, Garden ect.) does not appear in the list below the text field it has not been submitted to the Kandu database.
+Start filling in the first 3 letters of the title of the item (eg. Member, Garden, etc.) that you would like to reference in the form field. Select the relevant item from the list displayed below the text field. If the desired item (eg. Member, Garden ect.) does not appear in the list below the text field it has not been submitted to the Kandu database.
```

Comment:
As we've seen with our tests unsubmitted data does appear in the auto-lookup listing. Is this not the case? (referring to the last sentence)